### PR TITLE
Add class TaskManager + relevant methods to work with tasks edit, add

### PR DESCRIPTION
**Pull request by HoloborodkoM**

### CODE REVIEW

File: TaskManager.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+   constructor() { 
+      this.tasks = this.loadTasks();
+   }
+
+   loadTasks() {
+
+      const data = fs.readFileSync('tasks.json');
+      return JSON.parse(data);
+      
+   }
```
Comment (SergiiTkachuk) [inline TaskManager.js]: Better use "try catch" to optimization

File: tasks.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[{"id":"1686057362340","title":"df","description":"df","deadline":"sf","completed":false},{"id":"1686057399706","title":"123","description":"vongo","deadline":"2.3.1945","completed":false},{"id":"1686057483842","title":"rty","description":"dfg","deadline":"4.5.1345","completed":false},{"id":"1686057512013","title":"rth","description":"457","deadline":"3034-12-30","completed":false}]
```
Comment (SergiiTkachuk) [inline tasks.json]: Rewrite saveTasks() for better output

File: TaskManager.js (hunk 1 above)
Comment (HoloborodkoM) [reply TaskManager.js]: Done, Sir

File: tasks.json (hunk 2 above)
Comment (HoloborodkoM) [reply tasks.json]: Of course

**Pull request merged**